src/Submissions/Api/Areas/Admin/Controllers/BiobanksController.cs

```diff
@@ -0,0 +1,211 @@
+using Biobanks.Data.Entities;
+using Biobanks.Submissions.Api.Area.Admin.Models;
+using Biobanks.Submissions.Api.Constants;
+using Biobanks.Submissions.Api.Models.Emails;
+using Biobanks.Submissions.Api.Models.Shared;
+using Biobanks.Submissions.Api.Services.Directory;
+using Biobanks.Submissions.Api.Services.Directory.Contracts;
+using Biobanks.Submissions.Api.Services.EmailServices.Contracts;
+using Biobanks.Submissions.Api.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Biobanks.Submissions.Api.Area.Admin.Controllers;
```

Comment:
namespace doesn't match the folders now 😅
```suggestion
namespace Biobanks.Submissions.Api.Areas.Admin.Controllers;
```